### PR TITLE
Add/Remove/Modify files for Housekeeping

### PR DESCRIPTION
**Pull request by j-mes**
- Check for correct support link
- Change from Travis to Circle CI
- Change shield on README.md from Travis to Circle CI
- Move variables from /src/demos/config.json into origami.json
- Fix indentation from Spaces to Tabs
- Remove html files from /src/demos/
- Remove /src/demos/config.json
- Remove travis.yml
- Add circle.yml
- Passed OBT verify check
- Passed OBT runServer check

### CODE REVIEW

**Comment by onishiweb** [comment]
LGTM :+1:

File: origami.json [hunk 1]
```diff
@@ -1,37 +1,56 @@
...
\ No newline at end of file
+	"description": "Web font loader",
+	"origamiType": "module",
+	"origamiVersion": 1,
+	"support": "https://github.com/Financial-Times/o-fonts/issues",
+	"supportStatus": "active",
+	"ci": {
+		"circle": "https://circleci.com/api/v1/project/Financial-Times/o-fonts"
+	},
+	"options": {
```
Comment (AlbertoElias) [inline origami.json]: this is called demosDefaults

Comment (j-mes) [inline origami.json]: FYI: Fixed in the next commit.

**Comment by AlbertoElias** [comment]
:+1:

**Pull request merged**